pyfiglet/__init__.py

```diff
@@ -759,6 +767,28 @@ def getFonts(self):
         return self.Font.getFonts()
 
 
+def __color_to_ansi(color, isBackground):
```

Comment:
I don't think this function needs a `__` suffix, `color_to_ansi` would suffice.

Reply:
I made it "private" because it has nothing to do with the main functionality of the module so I didn't wanted to make that method visible